providers/yttcc/lib/config_variable_association.star

```diff
@@ -991,6 +991,39 @@ def get_vsphere_vars():
         vars["additionalFQDN"] = data.values["VSPHERE_ADDITIONAL_FQDN"].replace(" ", "").split(",")
     end
 
+    gpu = {}
+    gpuControlPlane = {}
+    if data.values["VSPHERE_WORKER_PCI_DEVICES"] != None:
```

Comment:
wait is this supposed to be `VSPHERE_CONTROL_PLANE_PCI_DEVICES` ?

Reply:
Fixed